### ISSUE TITLE
Include parse-latest.js within a dist/ directory of the repo

### ISSUE THREAD

**Issue by tomardern** [ParsePlatform/Parse-SDK-JS#26]
Hi,

I'm currently developing a method to carry out Unit tests on Cloud Code (Something I will probably open source), and was utilising parse-latest.js to include Parse specific functions.

Each time the repo is updated, are you able to include pre-built "parse-latest.js" files for React/Node/Browser, therefore users will not have the trouble of building the files through gulp each time?

Thanks,

**Comment by andrewimm** [comment]
Can you explain why you need a built copy for your unit tests? Are you running them in a browser, or in node?

Sounds like you could use the latest version of our SDK available from our Downloads page, or from our CDN: www.parsecdn.com/js/parse-latest.js

**Comment by andrewimm** [comment]
@simonbengtsson moving discussion here.

I actively use npm as a frontend package manager, and I believe developers who do the same will be using tools like Browserify or Webpack to compile their projects. If we were to include a compiled version in the npm module, how would you use it in a project? I'd like to understand this use case better.

**Comment by simonbengtsson** [comment]
Sure! I think that with the release of npm 3 many more people will start using npm as a front end package manager. One use case would be to upload the entire `node_packages` folder and reference scripts like `<script src="node_modules/jquery/dist/jquery.js"></script>`. I have seen this often with bower packages. The use case I had however was that I didn't use browserify and simply wanted to move all plain browser javascript dependencies to my dist folder.

In short provide the same functionality as repos like this one https://github.com/rabidaudio/bower-parse.

**Comment by andrewimm** [comment]
I see. We used to ship both lib/ and dist/ for our Parse+React package, but removed it after a few months as it seemed cluttered. I can understand your argument though, and we definitely don't want to stand in the way of your development. It's no harder to enable this, since we already produce the file at publish time. I'll add an entry to the `"files"` field on `package.json`

**Comment by andrewimm** [comment]
Included in 1.6.4, which just went to npm

**Comment by tomardern** [comment]
I'm using Karma to carry out unit tests on my Parse Code. As it runs within a browser environment, it's much easier and simpler to inject Parse-latest.js into the page rather than compiling from source.

Thanks for sorting this :)

**Comment by simonbengtsson** [comment]
I have been really impressed with the parse team. You guys rock, and thanks for this!
